gdal/ogr/ogrsf_frmts/GNUmakefile

```diff
@@ -1,7 +1,7 @@
 
 include ../../GDALmake.opt
 
-SUBDIRS-yes = generic geojson kml mem mitab vrt $(OGR_FORMATS)
+SUBDIRS-yes = generic geojson kml mem mitab vrt flatgeobuf $(OGR_FORMATS)
```

Comment:
only mandatory formats for a minimal build should be specified here. so flatgeobuf should not

Reply:
Ok will try to make the adjustments to move it into an optional format. Out of curiosity, why are geojson, kml and mitab mandatory?

Reply:
geojson and kml because of OGR_G_ExportToGeoJSON/OGR_G_ExportToKML
MITAB because its used by generic attribute indexing code